views/content/dao.md

```diff
@@ -128,7 +128,6 @@ The way this particular democracy works is that it has an **Owner** which works
                members[id] = Member({member: targetMember, memberSince: now, name: memberName});
             } else {
                 id = memberId[targetMember];
-                Member m = members[id];
```

Comment:
Do not remove this line. Instead I believe line 128 should be moved to 132. Need to test